custom/icds_reports/ucr/expressions.py

```diff
@@ -133,7 +139,11 @@ def __call__(self, item, context=None):
 
     def _get_forms(self, case_id, from_date, to_date, context):
         domain = context.root_doc['domain']
-        cache_hash = "{}{}{}".format(from_date.toordinal(), to_date.toordinal(), self.count)
+        cache_hash = "{}".format(self.count)
+        if from_date:
+            cache_hash += "{}".format(from_date.toordinal())
+        if to_date:
+            cache_hash += "{}".format(to_date.toordinal())
```

Comment:
Good catch. originally i did hashing because xmlns is a list of strings and lists can't be used in keys, so now just make them tuples: https://github.com/dimagi/commcare-hq/pull/15513/commits/c8caa5802f72640b08165ae7189228507bf74ca6